### DISCUSSION TITLE
Split bridge speed to bridges and internal bridges (and cooling too)

### DISCUSSION THREAD

**Discussion by Chodec80** [OrcaSlicer/OrcaSlicer#466] [Ideas]
Hello everyone,
I am just trying this amazing slicer and figured out two missing features. It would be nice to have individual speed setting for proper bridges and for internal bridges (first layer above infill). Would be also fine to separate cooling setting for bridges and internal bridges. I believe that everybody knows Superslicer and I enjoy these function there.

**Comment by SoftFever** [comment]
It makes sense to separate speed for internal and external bridges.
Do you mind to create a FR ticket?
